zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/Endpoint.kt

```diff
@@ -77,7 +76,7 @@ class Endpoint internal constructor(
       callbackName: String
     ) {
       val handler = inboundHandlers[instanceName] ?: error("no handler for $instanceName")
-      CoroutineScope(EmptyCoroutineContext).launch(dispatcher) {
+      CoroutineScope(dispatcher).launch {
```

Comment:
Very skeptical of both the before and after here, once again regarding naked scopes.

Reply:
I’m trying and failing to post work to the dispatcher. Best way to do that?